tools/test-harnesses/src/queries/index.ts

```diff
@@ -0,0 +1 @@
+export * from './lgId';
```

Comment:
nit: should this be `byLgId`?

Reply:
i was following `dom-testing-library` but if you feel strongly i can append `by`

Reply:
Nah, if that's an established pattern, it's fine